packages/opentelemetry/README.md

```diff
@@ -23,6 +25,10 @@ await init({
 
 This will automatically instrument a variety of commonly-used Node packages.
 
+When using code from the OpenTelemetry libraries, make sure you import it via `@prairielearn/opentelemetry` instead of installing it separately to ensure that there is only one version of each OpenTelemetry package in use at once. If the desired functionality is not yet exported, please add it!
```

Comment:
What would it look like if we wanted to directly report some other metric in our application code, for example using a plain Counter?

Reply:
Heh, I was waiting to document that until we decided whether or not we'd use a `Map` to store metrics, as that would change both the API and what ends up being documented. I'll add that before merging.

Reply:
Awesome, thanks!

Reply:
I added some more documentation in fe99a61, how does that look?

Reply:
Looks super easy to use. Nice!